src/Twilio/Rest/Client.php

```diff
@@ -220,9 +223,12 @@ public function request(string $method, string $uri, array $params = [], array $
         $logLevel = (getenv('DEBUG_HTTP_TRAFFIC') === 'true' ? 'debug' : $this->getLogLevel());
 
         $headers['User-Agent'] = 'twilio-php/' . VersionInfo::string() .
-                                 ' (PHP ' . PHP_VERSION . ')';
+                                 ' (' . php_uname("s") . ' ' . php_uname("m") . ')' .
+                                 ' PHP/' . PHP_VERSION;
         $headers['Accept-Charset'] = 'utf-8';
 
+        $headers['User-Agent'] .= ' ' . implode(' ', $this->userAgentExtensions);
```

Comment:
nit: Should check that there are any agent extensions before appending, or the header will get an extra space on the end.

Reply:
Noted. Will add an if-statement to verify it